packages/apps/reputation-oracle/server/docker-compose.yml

```diff
@@ -45,7 +42,7 @@ services:
         condition: service_healthy
     entrypoint: >
       /bin/sh -c "
-      /usr/bin/mc config host add myminio http://minio:9000 ${S3_ACCESS_KEY} ${S3_SECRET_KEY};
+      /usr/bin/mc config host add myminio http://minio:9000 minioadmin minioadmin;
```

Comment:
This and removal of env variables need to easily work with it out of the box, aligned with job launcher